dnf5.spec

```diff
@@ -632,6 +635,7 @@ Libdnf5 plugin that installs repository's Appstream data, for repositories which
 Summary:        Libdnf5 plugin for detecting and removing expired PGP keys
 License:        LGPL-2.1-or-later
 Requires:       libdnf5%{?_isa} = %{version}-%{release}
+Recommends:     gnupg2
```

Comment:
Can the plugin work without `gnupg2`? If not, I suggest changing this from  weak dependency to requirement.

Reply:
The plugin itself can be loaded without the package, but if the `gpg` command is not available on the system, it won't basically do anything. I recall some discussion around that, possibly some other provider of `gpg` command being possibly deployed, but right now I cannot find any of those. I can eventually change it to hard dep...

Reply:
In that case we can use `Requires: /usr/bin/gpg` which would be satisfied by any provider of the binary. But since there is none other and we cannot be sure that alternative version would work for our use-case, I would prefer keeping `Requires: gnupg2`.

I do not see much sense in ability to load a plugin which will do nothing. I find it confusing for the users.

Reply:
Thanks, I agree. Will change this to `Requires: gnupg2`.